tic_tac_toe/game.py

```diff
@@ -47,7 +59,13 @@ def _is_winning_combination(board, combination, player):
     Returns True of all three positions in the combination belongs to given
     player, False otherwise.
     """
-    pass
+    char_list = []
+    for position in combination:
+        char_list.append(board[position[0]][position[1]])
+    if len(set(char_list)) > 1 or player not in set(char_list):
+        return False
+    return True
```

Comment:
Wow yeah that would be quite a bit simpler! Basically my brain has a function called convolute:

def convolute(simplest_approaches):
   While True:
      for approach in simplest_approaches:
            don't do approach
       do something convoluted